src/pages/trading-specification/components/_elements.tsx

```diff
@@ -226,43 +230,52 @@ type TTableCellGroup = {
 }
 export const TableCellGroup = ({ data, market }: TTableCellGroup) => {
     const { is_row } = useRegion()
-    const symbol = data.symbol
-    const text = data.instrument
-    const dl_icon = !is_row ? data.dl_icon : ''
-    const [showPopUp, setShowPopUp] = useState(false)
+    const { symbol, instrument: text, dl_icon, swf_icon } = data
+    const [show_popUp, setShowPopUp] = useState(false)
+    const [popup_type, setPopupType] = useState<TPopupType>()
     const [is_mobile] = useBrowserResize(768)
 
+    const getStyledImg = (width, icon, type) => {
+        return is_mobile ? (
+            <StyledImg
+                src={icon}
+                width={width}
+                height="24px"
+                onTouchStart={() => {
+                    setShowPopUp(true)
+                    setPopupType(type)
+                }}
+            />
+        ) : (
+            <StyledImg
+                src={icon}
+                width={width}
+                height="24px"
+                onClick={() => {
+                    setShowPopUp(true)
+                    setPopupType(type)
+                }}
+            />
+        )
+    }
+
     if (data !== undefined)
         return (
             <CellIcon>
                 {symbol && <img src={icons[symbol]} width="24px" height="24px" />}
                 <StyledHeaderText type="small" align="start" as="p">
                     {text}
                 </StyledHeaderText>
-                {dl_icon &&
-                    is_row &&
-                    (is_mobile ? (
-                        <img
-                            src={dl}
-                            width="24px"
-                            height="24px"
-                            onTouchStart={() => {
-                                setShowPopUp(true)
-                            }}
-                        />
-                    ) : (
-                        <img
-                            src={dl}
-                            width="24px"
-                            height="24px"
-                            onClick={() => {
-                                setShowPopUp(true)
-                            }}
-                        />
-                    ))}
-                {showPopUp && (
+                {is_row ? (
+                    <>
+                        {/* {dl_icon && getStyledImg('24px', dl, 'dl')} */}
+                        {swf_icon && getStyledImg('30px', swf, 'swf')}
+                    </>
+                ) : null}
```

Comment:
```suggestion
                {is_row && (
                        {swf_icon && getStyledImg('30px', swf, 'swf')}
                )}
```

Reply:
I commented this line instead of removing because `dl_icon` may require in the next release.